lib/ktsllex/schemas.ex

```diff
@@ -97,16 +103,27 @@ defmodule Ktsllex.Schemas do
   end
 
   defp update_connect_name(:error, _schema_name, _type), do: :error
-  defp update_connect_name(schema, _schema_name, "-value"), do: schema
+  defp update_connect_name(schema, _schema_name, "value"), do: schema
 
-  defp update_connect_name(schema, schema_name, "-key") do
+  defp update_connect_name(schema, schema_name, "key") do
     schema
     |> Map.put("connect.name", schema_name <> ".Key")
   end
 
   defp read_schema(base_schema_file, type) do
-    (base_schema_file <> type <> ".json")
-    |> FileJson.read!()
+    json_file =
+      (base_schema_file <> ".json")
+      |> FileJson.read!()
+
+    case json_file do
```

Comment:
I'd suggest this is moved into its own function, maybe `extract_schema`, then this function does not need the `type`, and `extract_schema` can be put in the pipeline right after this.
```elixir
defp extract_schema(%{"key_avro_schema" => key_schema, "value_avro_schema" => value_schema}, type) do
...
```